packages/elements/src/slider/__demo__/index.html

```diff
@@ -102,6 +102,33 @@
       <ef-slider range min="0.1" max="10" value="0" step="0.5" show-steps show-input-field></ef-slider>
     </demo-block>
 
+    <demo-block layout="normal" header="Custom label">
```

Comment:
similar to doc, We might want to add more examples such as no-label markers & non-text content e.g. ef-icon as a marker label.

Reply:
updated